assembly/src/tests.rs

```diff
@@ -249,19 +249,10 @@ fn comment_in_nested_control_blocks() {
     assert_eq!(expected, format!("{}", script));
 }
 
-#[test]
-fn comment_before_script() {
-    let assembler = super::Assembler::new();
-    let source = " # starting comment # begin push.1 push.2 add end";
-    let script = assembler.compile_script(source).unwrap();
-    let expected = "begin span pad incr push(2) add end end";
-    assert_eq!(expected, format!("{}", script));
-}
-
```

Comment:
I think we still want to test for comments that come before the script. Can you restore this test and update it to work with the new format?